docs/source/trainer.rst

```diff
@@ -148,14 +148,26 @@ So you can run it like so:
 
 ------------
 
+Validation
+----------
+After training, you can perform a new evaluation epoch over the validation set
+with :meth:`pytorch_lightning.trainer.trainer.Trainer.validate`. This might be
+useful if you want to collect new metrics from a model you already trained.
```

Comment:
only possible after training?

Reply:
Not necessarily, indeed. What I wanted to clarify is that you don't need this method in the normal training loop, it's only for performing validation on its own. Maybe this is better?

```suggestion
You can perform an evaluation epoch over the validation set, outside of the training loop,
using :meth:`pytorch_lightning.trainer.trainer.Trainer.validate`. This might be
useful if you want to collect new metrics from a model that has already been trained.
```

Reply:
updated a bit. Might solve this issue https://github.com/PyTorchLightning/pytorch-lightning/issues/4170

Reply:
Wonderful